src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1358,4 +1358,10 @@ I agree to all of the foregoing:</value>
   <data name="Naming_rules" xml:space="preserve">
     <value>Naming rules</value>
   </data>
+  <data name="Enable_power_save_mode" xml:space="preserve">
+    <value>Enable power save mode (disables all background analysis)</value>
```

Comment:
this feels like something that should be part of the mainline VS UI.

Reply:
Yes, the idea is to start with just Roslyn features, and then based on user feedback move it to a platform layer option in future.

Reply:
gotcha.

Reply:
Note: by "mainline vs ui" i mean: easy to click a button in the VS ui, not have to go into tools|options :)

Reply:
➡️ We'll leave final placement to the platform team